playground/terraform/README.md

```diff
@@ -16,74 +16,118 @@
+* `state.tfbackend` environment variables:
 ```
-
-## 3. Deploy application
-
-To deploy application run following steps:
-
-* Authinticate in Artifact registry
-
-```bash
-gcloud auth configure-docker us-central1-docker.pkg.dev
+bucket               = "bucket_name"         #input bucket name - will be used for terraform tfstate file
 ```
+Then, configure authentication for the Google Cloud Platform:
 
-* Вeploy backend services
-
-```bash
-./gradlew playground:terraform:deployBackend -Pproject_environment="env-name" -Pdocker-tag="tag"
+* The following commands allows us to configure account and project
+```
+gcloud init
+```
 ```
+gcloud auth application-default login
+```
+# Infrastructure deployment:
+* To deploy the Payground infrastructure, use the following command (please be sure that you are in the "beam" folder):
+```
+./gradlew playground:terraform:InitInfrastructure -Pproject_environment="env" (env - folder name which you created for configuration files)
+```
+# Playground deployment:
+Afrer successfully deploying Playground infrastrcuture in the previous step, authenticate on Docker and Google Kubernetes Engine<br>
+The following command will authenticate us in the Docker registry
+```
+ gcloud auth configure-docker `chosen_region`-docker.pkg.dev
+```
+The following command will authenticate in GKE
+```
+gcloud container clusters get-credentials --region `chosen_location` `gke_name` --project `project_id`
+```
+Find a Static IP in your GCP project>VPC Network>IP Addresses>pg-static-ip
+<br>Add following DNS A records for the Static IP address:
+```
+* java.playground.zone
+* python.playground.zone
+* scio.playground.zone
+* go.playground.zone
+* router.playground.zone
+* playground.zone
+```
+Where "playground.zone" is your registered DNS zone<br>
+[More about DNS zone registration](https://domains.google/get-started/domain-search/)<br>
+[More about A records in DNS](https://support.google.com/a/answer/2579934?hl=en)
 
-* Deploy frontend service
+To deploy Beam Playground to the configured envrionment, please execute the following command (Ensure you are in the "beam" folder):
+```
+./gradlew playground:terraform:gkebackend -Pproject_environment="env" -Pdocker-tag="tag" -Pdns-name="playground.zone" (env - folder name which you created for configuration files, tag - image tag for backend, playground.zone - chosen DNS for Playground)
+```
+During script execution, a Google managed certificate will be created. [Provisioning might take up to 60 minutes](https://cloud.google.com/load-balancing/docs/ssl-certificates/google-managed-certs).
 
-```bash
-./gradlew playground:terraform:deployFrontend -Pproject_environment="env-name" -Pdocker-tag="tag" ```
+# Environment validation steps:
+1. Run "helm list" command in the console to ensure that status is "deployed":
+```
+NAME            NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                          APP VERSION
```

Comment:
Reduce empty space between "UPDATED" and "STATUS" to prevent horizontal scroll bar

Reply:
fixed